tests/unit/test_utils.py

```diff
@@ -25,7 +28,7 @@
 
 _EXCLUDED_DECORATOR_FUNCTIONS = ["wraps", "total_ordering", "abstractmethod"]
 
-_EXCLUDED_FUNCTIONS = ["jinja2_convenience_function", "import_module"]
+_EXCLUDED_FUNCTIONS = ["jinja2_convenience_function", "import_module", "get_network_driver"]
```

Comment:
Why is this function excluded?

Reply:
Its a method imported from napalm, we just use it. I don't see what it would offer as a filter.

Reply:
doh`